src/bentoml/_internal/utils/onnx.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
```

Comment:
should we move this file to `src/bentoml/_internal/frameworks/utils/onnx.py`?

Reply:
Can you also make a PR that move the tensorflow utils to this folder after this PR?

Reply:
That's what I'm thinking. I will move both to the same folder in another pr

Reply:
Is this ready to review?

Reply:
@aarnphm ready now